cmd/clusterctl/examples/openstack/generate-yaml.sh

```diff
@@ -186,14 +199,17 @@ if [[ "$PROVIDER_OS" == "coreos" ]]; then
 else
```

Comment:
you updated centos and ubuntu
but how about coreos ? I didn't see the change ,maybe a follow up??

Reply:
@jichenjc Thanks commenting.i added code for coreos in https://github.com/kubernetes-sigs/cluster-api-provider-openstack/commit/b06fd90d5eaff2c379e3322bf8bcbc5e3a60f3bc

Reply:
I Updated files for coreos.